docs/books/admin_guide/09-backups.md

```diff
@@ -144,7 +144,7 @@ The use of a naming convention allows one to quickly target the contents of a ba
 
 !!! Note
 
-    In the Linux world, except for a few exceptions in GUI environments (such as .jpg, .mp4, .gif), most files do not have the concept of extension. In other words, most file extensions are not required. The reason for artificially adding suffixes is to facilitate recognition by human users. If the systems administrator sees a `.tar.gz` or `.tgz` file extension, for instance, then he knows how to deal with the file.
+    In the Linux world, except for a few exceptions in GUI environments (such as .jpg, .mp4, .gif), most files do not have the concept of extension. In other words, most files do not require an extension. The reason for artificially adding suffixes is to facilitate recognition by human users. If the systems administrator sees a `.tar.gz` or `.tgz` file extension, for instance, then he knows how to deal with the file.
 
```

Comment:
@jimcat8 I'm going to do a review of each change, only if it needs to be edited by you. This rewording is fine.

Reply:
You can ignore this. After further review, I could not do a good review of your changes.